plugin/ball_plugin.cpp

```diff
@@ -0,0 +1,72 @@
+#include <gazebo/common/common.hh>
+#include <gazebo/gazebo.hh>
+#include <gazebo/physics/physics.hh>
+#include <gazebo/sensors/sensors.hh>
+
+#include "message/input/gazebo/Ball.pb.h"
+#include "nuclear_network.h"
+
+namespace gazebo {
+class NUbotsBallPlugin : public ModelPlugin {
+
+public:
+    // Inherit constructors
+    using ModelPlugin::ModelPlugin;
+
+    /**
+     * The load function is called by Gazebo when the plugin is inserted into simulation
+     * @param _model A pointer to the model that this plugin is attached to
+     * @param _sdf   A pointer to the plugin's SDF element
+     */
+    void Load(physics::ModelPtr _model, sdf::ElementPtr _sdf) {
+
+        // Output a debug line
+        gzdbg << "Attaching a ball plugin to model [" << _model->GetName() << "]" << std::endl;
```

Comment:
Should we bring in fmt::format?

Reply:
For one or two lines with no complex formatting? Probably not